source/core/query-plans.txt

```diff
@@ -128,7 +128,7 @@ changes to the plan cache.
 
 
 Query Plan and Cache Information
---------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I would suggest making this heading and the "plan cache flushes" heading h1s (meaning the `-` underline). It looks like that was the original intent based on the prior version of the copy.